### PR TITLE
FEATURE/TEXT FIELD

### PR DESCRIPTION
**Pull request by nanisaladi24**
This PR (pull req) is for adding new text field input UI.

### CODE REVIEW

File: Todo_list/todo_list/src/App.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function App() {
         <img src={logo} className="App-logo" alt="logo" />
         <p>
           {/* Edit <code>src/App.js</code> and save to reload. */}
-          Hello Naidu
```
Comment (vinnytalupula) [inline Todo_list/todo_list/src/App.js:12]: Why did you change

**Review by vinnytalupula — CHANGES_REQUESTED** [review]

File: Todo_list/todo_list/src/App.js (hunk 1 above)
Comment (nanisaladi24) [reply Todo_list/todo_list/src/App.js:12]: for fun

**Review by vinnytalupula — APPROVED** [review]
